chapter_multilayer-perceptrons/dropout_vn.md

```diff
@@ -215,7 +215,7 @@ Như ta mong muốn, kỳ vọng không bị thay đổi, hay nói cách khác $
 ## Dropout in Practice
 -->
 
-## Dropout trong thực tế
+## Dropout trong Thực hành
```

Comment:
mình thấy như cũ cũng ổn mà, mà `thực hành` lại là động từ
```suggestion
## Dropout trong Thực hành
```

Reply:
`Thực hành` cũng có thể coi là danh từ. Tuy nhiên mình chưa biết từ `in pratice` nhóm hay dịch như thế nào

Reply:
Tại `thực tế` mình có thể hiểu theo kiểu `in real life, in the wild,...`
Đoạn này ý chỉ học lý thuyết xong rồi giờ thực hành. Nếu muốn chuẩn danh từ thì có thể từ nữa là:
```suggestion
## Dropout trong Thực tiễn
```

Reply:
Yup nghe ổn đó